bot/src/main/java/com/ibdiscord/command/commands/HelpCommand.java

```diff
@@ -0,0 +1,35 @@
+package com.ibdiscord.command.commands;
+
+import com.ibdiscord.command.Command;
+import com.ibdiscord.command.CommandContext;
+import com.ibdiscord.command.permissions.CommandPermission;
+import com.ibdiscord.main.IBai;
+import net.dv8tion.jda.core.EmbedBuilder;
+import net.dv8tion.jda.core.Permission;
+
+import java.awt.*;
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class HelpCommand extends Command {
+
+    public HelpCommand() {
+        super("help",
+                new HashSet<>(),
+                CommandPermission.discord(),
+                new HashSet<>());
+    }
+    @Override
+    protected void execute(CommandContext context) {
+        String botPrefix = IBai.getConfig().getStaticPrefix();
```

Comment:
We'll need to get the guild specific prefix from the database as opposed to the static prefix. If we can't do this right now, later is fine. Can we get a TODO here?

Reply:
Yep sounds good.